pkg/vsphere/vm/vm.go

```diff
@@ -560,3 +564,41 @@ func (vm *VirtualMachine) DatastoreReference(ctx context.Context) ([]types.Manag
 	}
 	return mvm.Datastore, nil
 }
+
+// GetVCHUpdateStatus tells if an upgrade/update has already been started based on the UpdateInProgress flag in ExtraConfig
+// If error != nil, VCH upgrade/update will exit; so the value of the VCH upgrade/update status is not important
```

Comment:
method help should not talk about the behavior of callers
here should describe while any error happens in vm operation, will return error